Example/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - netfox (1.21.0)
-  - SmileID (10.0.0-beta02):
+  - SmileID (10.0.0-beta03):
```

Comment:
Shouldn't this be beta07?

Reply:
The feature branch is out of sync with main. Need to update that branch. Will do that in another PR.